src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_upscale.py

```diff
@@ -529,21 +525,7 @@ def prepare_latents(self, batch_size, num_channels_latents, height, width, dtype
         return latents
 
     def upcast_vae(self):
-        dtype = self.vae.dtype
```

Comment:
Hmm, should the deprecation be not called from here, too? I am seeing some methods where the deprecation call exists and some where it doesn't (for example, this one). Is this expected?